diff --git a/dist/esm/server-functions-handler.js b/dist/esm/server-functions-handler.js
index c493774211d1ea4b966e9fcd437e3e833d8f932e..69e7b1590fbdc9640b62152e1aaff7331bf7f01c 100644
--- a/dist/esm/server-functions-handler.js
+++ b/dist/esm/server-functions-handler.js
@@ -49,21 +49,25 @@ var handleServerAction = async ({ request, context, serverFnId }) => {
 				if (methodUpper === "GET") {
 					const payloadParam = url.searchParams.get("payload");
 					if (payloadParam && payloadParam.length > MAX_PAYLOAD_SIZE) throw new Error("Payload too large");
-					const payload = payloadParam ? parsePayload(JSON.parse(payloadParam)) : {};
-					payload.context = safeObjectMerge(payload.context, context);
-					payload.method = methodUpper;
-					return await action(payload);
+					const payload = payloadParam ? parsePayload(JSON.parse(payloadParam)) : void 0;
+					return await action({
+						data: payload?.data,
+						context: safeObjectMerge(payload?.context, context),
+						method: methodUpper
+					});
 				}
 				let jsonPayload;
 				if (contentType?.includes("application/json")) jsonPayload = await request.json();
-				const payload = jsonPayload ? parsePayload(jsonPayload) : {};
-				payload.context = safeObjectMerge(payload.context, context);
-				payload.method = methodUpper;
-				return await action(payload);
+				const payload = jsonPayload ? parsePayload(jsonPayload) : void 0;
+				return await action({
+					data: payload?.data,
+					context: safeObjectMerge(payload?.context, context),
+					method: methodUpper
+				});
 			})();
-			const unwrapped = res.result || res.error;
+			const unwrapped = res.error !== void 0 ? res.error : res.result;
 			if (isNotFound(res)) res = isNotFoundResponse(res);
-			if (!isServerFn) return unwrapped;
+			if (!isServerFn && (unwrapped instanceof Response || unwrapped === null || typeof unwrapped !== "object")) return unwrapped;
 			if (unwrapped instanceof Response) {
 				if (isRedirect(unwrapped)) return unwrapped;
 				unwrapped.headers.set(X_TSS_RAW_RESPONSE, "true");
