diff --git a/src/transform/segments.js b/src/transform/segments.js
index 3433a52004d3c4c3f71beea109d0fa7939fee974..3bbc994983e261b97a17b809bc866f69ea456166 100644
--- a/src/transform/segments.js
+++ b/src/transform/segments.js
@@ -517,6 +517,11 @@ export function convert_source_map_to_mappings(
 	function set_bracket_computed_mapping(node, mappings) {
 		if (has_location(node.key)) {
 			const key = node.key;
+			const start_key = `${key.loc.start.line}:${key.loc.start.column - 1}`;
+			const end_key = `${key.loc.end.line}:${key.loc.end.column + 1}`;
+			if (!src_to_gen_map.get(start_key)?.length || !src_to_gen_map.get(end_key)?.length) {
+				return;
+			}
 			mappings.push(
 				get_mapping_from_node(
 					/** @type {AST.NodeWithLocation} */ ({
