diff --git a/dist/chunk-mocker.js b/dist/chunk-mocker.js
--- a/dist/chunk-mocker.js
+++ b/dist/chunk-mocker.js
@@ -515,10 +515,10 @@
 		const config = this.config;
 		const fsRoot = join("/@fs/", config.root);
 		// URL can be /file/path.js, but path is resolved to /file/path
-		if (path.startsWith(config.root)) {
+		if (path === config.root || path.startsWith(config.root + "/")) {
 			return path.slice(config.root.length);
 		}
-		if (path.startsWith(fsRoot)) {
+		if (path === fsRoot || path.startsWith(fsRoot + "/")) {
 			return path.slice(fsRoot.length);
 		}
 		return path;
