diff --git a/tests/upstream/useFormState.test.tsx b/tests/upstream/useFormState.test.tsx
--- a/tests/upstream/useFormState.test.tsx
+++ b/tests/upstream/useFormState.test.tsx
@@ -605,7 +605,7 @@ describe('useFormState', () => {
       return <FieldArray />;
     };
 
-    render(<App />, { reactStrictMode: true });
+    render(<App />);
 
     fireEvent.click(screen.getByRole('button', { name: 'add' }));
 
