diff --git a/tests/upstream/_types/form.test-d.ts b/tests/upstream/_types/form.test-d.ts
--- a/tests/upstream/_types/form.test-d.ts
+++ b/tests/upstream/_types/form.test-d.ts
@@ -7,8 +7,8 @@ import type {
   FieldValues,
   Resolver,
   UseFormGetFieldState,
-} from '../../src/types';
-import { useForm } from '../../src/useForm';
+} from '../../../src/types';
+import { useForm } from '../../../src/useForm';
 
 import type { Equal, Expect } from './__fixtures__';
 
