diff --git a/tests/upstream/_types/use-form-context.test-d.ts b/tests/upstream/_types/use-form-context.test-d.ts
--- a/tests/upstream/_types/use-form-context.test-d.ts
+++ b/tests/upstream/_types/use-form-context.test-d.ts
@@ -1,4 +1,4 @@
-import { useFormContext } from '../../src/useFormContext';
+import { useFormContext } from '../../../src/useFormContext';
 
 import type { Equal, Expect } from './__fixtures__';
 
